src/python/pants/option/arg_splitter_test.py

```diff
@@ -222,11 +222,11 @@ def test_descoping_qualified_flags(self) -> None:
 
     def test_passthru_args(self) -> None:
         self.assert_valid_split(
```

Comment:
FWIW: There were no issues with the `ArgSplitter` class, but this change to the test helped to confirm that.